.env

```diff
@@ -1,10 +1,10 @@
 AMQP_URI=amqp://localhost:5672
-ARITHMETIC_CIRCUIT_SIZE=16..23
-BYTE_PACKING_CIRCUIT_SIZE=9..21
-CPU_CIRCUIT_SIZE=12..25
-KECCAK_CIRCUIT_SIZE=14..20
-KECCAK_SPONGE_CIRCUIT_SIZE=9..15
-LOGIC_CIRCUIT_SIZE=12..18
-MEMORY_CIRCUIT_SIZE=17..28
-MEMORY_BEFORE_CIRCUIT_SIZE=7..23
-MEMORY_AFTER_CIRCUIT_SIZE=7..27
+ARITHMETIC_CIRCUIT_SIZE=16..21
+BYTE_PACKING_CIRCUIT_SIZE=8..21
+CPU_CIRCUIT_SIZE=10..21
+KECCAK_CIRCUIT_SIZE=4..20
+KECCAK_SPONGE_CIRCUIT_SIZE=8..17
+LOGIC_CIRCUIT_SIZE=4..21
+MEMORY_CIRCUIT_SIZE=17..24
+MEMORY_BEFORE_CIRCUIT_SIZE=16..23
+MEMORY_AFTER_CIRCUIT_SIZE=7..23
```

Comment:
The changes are unrelated, but the previous ones were somehow outdated. Namely:

- CPU can't get lower than 2^10 because of `init`
- MEM_BEFORE can't get lower than 2^16 because of Kernel code + shift table
- MEM_AFTER cannot get higher than MEM_BEFORE
- Adjusting lowest bounds, either capped by MerkleCap config or Range-Check size.
- Finally, experiments seem to indicate we perform better with CPU sizes around 2^19, hence skipping ranges above 2^20 and adjusting related tables that are bounded by CPU.